server/src/main/java/io/crate/expression/tablefunctions/EmptyRowTableFunction.java

```diff
@@ -43,7 +42,7 @@ public class EmptyRowTableFunction {
 
     public static void register(Functions.Builder builder) {
         builder.add(
-            Signature.table(NAME, RowType.EMPTY.getTypeSignature()).withFeature(Scalar.Feature.NON_NULLABLE),
+            Signature.table(NAME, RowType.EMPTY.getTypeSignature()),
```

Comment:
All table functions were marked as `NON_NULLABLE` but unmarking them all. It fits the definition but there is no real benefit from Lucene query building.